package.json

```diff
@@ -3,22 +3,21 @@
   "version": "1.0.0",
   "scripts": {
     "build": "vite build",
-    "lint": "eslint --cache .",
+    "lint": "eslint . --ext .ts",
```

Comment:
Added the explicit extensions option since it wasn't always capturing all files.